PhysicsTools/PatAlgos/interface/MuonMvaEstimator.h

```diff
@@ -0,0 +1,46 @@
+#ifndef __PhysicsTools_PatAlgos_MuonMvaEstimator__
+#define __PhysicsTools_PatAlgos_MuonMvaEstimator__
+#include "DataFormats/MuonReco/interface/Muon.h"
+#include "DataFormats/PatCandidates/interface/Muon.h"
+#include "DataFormats/BTauReco/interface/JetTag.h"
+#include "TMVA/Reader.h"
+
+namespace reco {
+  class JetCorrector;
+}
+namespace pat {
+  class MuonMvaEstimator{
+  public:
+    MuonMvaEstimator();
+    void initialize(std::string weightsfile, 
```

Comment:
```initialize(std::string const& weightsfile, ```

Reply:
It's called one in the PATMuonProducer::produce. It's a waste of time to change it.